js/main.js

```diff
@@ -77,25 +105,50 @@ function toggleNoEntries() {
   }
 }
 
-const $anchorTag = document.querySelector('.anchor-tag');
 $anchorTag.addEventListener('click', function () {
   viewSwap('entries');
 }
 );
 
-const $anchorTagTwo = document.querySelector('.new-button');
 $anchorTagTwo.addEventListener('click', function () {
   viewSwap('entry-form');
+  const $notes = document.querySelector('#notes');
+  $notes.innerHTML = '';
 }
 );
 
 function viewSwap(string) {
   if (string === 'entries') {
     $entries.classList.remove('hidden');
     $entryForm.classList.add('hidden');
-  } else if (string === 'entry-form') {
+  } else
+  if (string === 'entry-form') {
     $entries.classList.add('hidden');
     $entryForm.classList.remove('hidden');
   }
   data.view = string;
 }
+
+// This is to swap pages if icon is clicked
+$list.addEventListener('click', function () {
+  if (event.target.tagName === 'I') {
+    viewSwap('entry-form');
```

Comment:
This should be called at the end of the function, after all the inputs get filled in